_data/participants_meeting_2019-06.yml

```diff
@@ -10,6 +10,9 @@
 - name: Wolfram Decker
   affiliation: TU Kaiserslautern
 
+- name: Christian Eder
+  affiliation: University of Leipzig
+
```

Comment:
Intentional?

Reply:
Absolutely. Christian will come to the meeting and is now in Leipzig. I was just to lazy to make 2 commits and PR's. Do you prefer me to make two commits out of those?